plugins/nemo-scaled-evals/examples/tasks/hello-world/task/instruction.md

```
@@ -0,0 +1,4 @@
+<!-- SPDX-FileCopyrightText: Copyright (c) 2025-2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+Create `/app/hello.txt` containing exactly `hello world` followed by a newline.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)